akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -11,6 +11,7 @@ import akka.serialization.{ Serialization, SerializationExtension }
 import akka.stream._
 import akka.stream.stage.{ GraphStage, GraphStageLogic, InHandler, OutHandler }
 import akka.util.OptionVal
+import akka.remote.RemoteMetricsExtension
```

Comment:
and it is marked as internal API, so sounds like we could just not add it for artery